sqld/src/error.rs

```diff
@@ -53,6 +56,16 @@ pub enum Error {
     Anyhow(#[from] anyhow::Error),
     #[error("Invalid host header: `{0}`")]
     InvalidHost(String),
+    #[error("Namespace `{0}` doesn't exist")]
+    UnexistingNamespace(String),
```

Comment:
```suggestion
    NamespaceDoesntExist(String),
```

This reads a bit better